cmake/Modules/FindLIBMODULEMD.cmake

```diff
@@ -0,0 +1,40 @@
+# Try to find libmodulemd 2.3+
```

Comment:
is there any reason to not just use PKG_SEARCH_MODULE directly in CMakeLists.txt?

Reply:
Honestly, I knew nothing about CMake prior to writing this patch. I just copied the GLib2 approach and trimmed it down to only what we needed from libmodulemd. If that's a sufficient answer (and preferred), I can whittle it down further.